### ISSUE TITLE
vergelijk teksten uit test-set met tekst uit training-set

### ISSUE THREAD

**Issue by bvdbroekrvo** [bvdbroekrvo/sdg_classifier#4]
Check of teksten uit test set (semantisch) overeenkomen met teksten uit training set, dus dat projecten verschillende ids hebben maar zowel voorkomen in test als trainingsdata.
